__tests__/app.test.js

```diff
@@ -99,9 +99,44 @@ describe("app", () => {
     })
   })
  });
+ describe("GET /api/articles", () => {
+  test("200: responds with a status of 200 when article objects are found", () => {
+    return request(app).get(`/api/articles`).expect(200);
+  });
+  test("200: responds with correct comment_count on related actricle_id", () => {
+    return request(app)
+    .get(`/api/articles`)
+    .expect(200)
+    .then((response) => {
+      const articleIdToTest = 34;
+      const article = response.body.articles.filter(article => article.article_id === articleIdToTest);
+      const expectedCommentCount = 11;
```

Comment:
I'm not sure if this test block is unfinished or if you were trying to do something global but  it isn't really doing anything at the moment as there is no assertion